code/modules/clothing/glasses/hud.dm

```diff
@@ -58,4 +58,13 @@
 	vision_flags = SEE_MOBS
 	see_invisible = SEE_INVISIBLE_OBSERVER_NOLIGHTING // the define name is just misleading, don't worry
 	toggleable = 0
-	actions_types = list()
\ No newline at end of file
+	actions_types = list()
+
+/obj/item/clothing/glasses/hud/xenohud
+	name = "XenoMate HUD"
+	desc = "A heads-up display that scans any nearby xenomorph's data."
+	icon_state = "securityhud"
+	flags_armor_protection = FALSE
+	toggleable = TRUE
+	hud_type =MOB_HUD_XENO_STATUS
```

Comment:
`flags_armor_protection = 0` until we get a proper `NONE` define.
Put a space between `=` and `MOB_HUD_XENO_STATUS`.
All good otherwise.